flang/test/Lower/intrinsic-procedures/get_command_argument.f90

```diff
@@ -0,0 +1,115 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck --check-prefixes=CHECK,CHECK-32 -DDEFAULT_INTEGER_SIZE=32 %s
+! RUN: flang-new -fc1 -fdefault-integer-8 -emit-fir %s -o - | FileCheck --check-prefixes=CHECK,CHECK-64 -DDEFAULT_INTEGER_SIZE=64 %s
+
+! CHECK-LABEL: func @_QPget_command_argument_test1(
+! CHECK-32-SAME: %[[num:.*]]: !fir.ref<i32>) {
+! CHECK-64-SAME: %[[num:.*]]: !fir.ref<i64>) {
+subroutine get_command_argument_test1(num)
```

Comment:
[nit] I think that you could skip `get_command_argument` in subroutine names. The filename already tells us which intrinsic is being tested. This would reduce the "noise" a bit. Instead, I would rename tests with something that would highlight their uniqueness. For example: `subroutine number_only` and then `subroutine number_and_length)`. Or something similar :)

Reply:
Done. What do you think about the test names now?